### DISCUSSION TITLE
MicroPython image for OLIMEX-H405

### DISCUSSION THREAD

**Discussion by mcblack** [micropython/micropython#15683] [STM32 / Pyboard]
Hello *,

in 2021, I was able to adapt the pyboard description files for a similar OLIMEX-H405 board and got micropython running. But an update to the recent micropython interpreter version is not working. I adapted all relevant files and can build the image. But it seems not to start probably on the board. There is no virtual COM Port and also no flash drive...

I attached my board description here: 
[OLIMEX_H405.zip](https://github.com/user-attachments/files/16675563/OLIMEX_H405.zip)

May be someone can help and find my mistake - or may be you could provide an idea how to debug this in the CubeMXIDE environment?

thanks in advance
  Mario

**Comment by IHOXOHI** [comment]
The H407 works correctly.
